cmd/root.go

```diff
@@ -50,15 +60,19 @@ var RootCmd = &cobra.Command{
 	Use:     "gh-metrics",
 	Short:   "gh-metrics: provide summary pull request metrics",
 	Version: Version,
-	Run: func(cmd *cobra.Command, args []string) {
-		owner, _ := cmd.Flags().GetString("owner")
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
By using `RunE`, cobra will exit nonzero with any non-`nil` `error` returned by the func.